browser/ui/views/toolbar/brave_app_menu.cc

```diff
@@ -38,3 +49,39 @@ MenuItemView* BraveAppMenu::AddMenuItem(views::MenuItemView* parent,
 
   return menu_item;
 }
+
+void BraveAppMenu::RecordMenuUsage(int command_id) {
+  misc_metrics::MenuGroup group;
+
+  switch (command_id) {
+    case IDC_NEW_WINDOW:
+    case IDC_NEW_TAB:
+    case IDC_NEW_INCOGNITO_WINDOW:
+    case IDC_NEW_OFFTHERECORD_WINDOW_TOR:
+      group = misc_metrics::MenuGroup::kTabWindow;
+      break;
+    case IDC_SHOW_BRAVE_WALLET:
+    case IDC_SHOW_BRAVE_SYNC:
+    case IDC_SHOW_BRAVE_REWARDS:
+      group = misc_metrics::MenuGroup::kBraveFeatures;
```

Comment:
perhaps we can add a comment to "brave/app/brave_command_ids.h" around these entries saying that if anything similar is added, it should be reflected in this function?